mia/commands/definition.py

```diff
@@ -31,7 +31,15 @@
 import sys
 import zipfile
 import distutils.dir_util
-from urllib.request import urlretrieve
+
+try:
+    # For Python 3.
+    from urllib.request import urlretrieve
+except ImportError:
+    # For Python 2.
+    from urllib import urlretrieve
```

Comment:
Thoughts on getting rid of this in favour of shelling out to curl or wget? I'm not sure I see the benefit of using python for downloading, as it seems we'll require more code to get to a place where we're not redownloading each time, and where we can continue partial downloads. We get this in one line with  `wget --continue <...>`, as it's very consistent across *nix platforms

I've alluded to my feelings in a few places, but don't have a sense of how you feel about that :)